src/extension.ts

```diff
@@ -12,7 +12,7 @@ import { GenerateBugReportCommand } from './commands/generateBugReport';
 import { ModuleCallsDataProvider } from './providers/moduleCalls';
 import { ModuleProvidersDataProvider } from './providers/moduleProviders';
 import { ServerPath } from './utils/serverPath';
-import { config, handleLanguageClientStart } from './utils/vscode';
+import { config, handleLanguageClientStart as handleLanguageClientStartError } from './utils/vscode';
```

Comment:
Nit: What do you think about renaming the function to `handleLanguageClientStartError` and not using an alias here?

Reply:
I wonder why the rename quick action did that?